docs/regenerating_the_database.md

```diff
@@ -10,6 +10,8 @@ Steps for regenerating the database:
  - Enter rails console using rails c -e production in separate terminal
  - Delete all Institution objects in the database using Institution.delete_all
  - Start the job to create the institution types by running the CreateInstitutionTypesJob.new.perform_now(). Wait for it to finish.
+- Start the job to create the Subjects by running the CreateSubjectsJob.new.perform_now(). Wait for it to finish.

```

Comment:
Pls add these commands also to regeneration script

Reply:
The regeneration script is not present on this branch because it had been branched out before the regeration script branch was merged into master.